saleor/order/models.py

```diff
@@ -383,7 +383,9 @@ def __str__(self):
 
 class OrderNote(models.Model):
     user = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+        settings.AUTH_USER_MODEL, blank=True, null=True,
+        verbose_name=pgettext_lazy('Order note field', 'user'),
```

Comment:
We have an issue (#1465) for removing verbose names from models in favor of explicit labels in forms, so we could also remove this `verbose_name`.